draft-ietf-rtcweb-jsep.xml

```diff
@@ -1977,13 +1977,224 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
+              as described in the
+              <xref target="sec.parsing-a-desc"/> section below. If parsing
+              fails for any reason, processing MUST stop and an error MUST be
+              returned.
+            </t>
+            <t>Finally, the parsed SessionDescription is applied as described in
+              the <xref target="sec.applying-a-local-desc"/> section below.</t>
+          </list>
+        </t>
+      </section>
 
-      <section title="Parsing an Answer" anchor="sec.parsing-an-answer"></section>
+      <section title="Processing a Remote Description" anchor="sec.processing-a-remote-desc">
+        <t>When a SessionDescription is supplied to setRemoteDescription, the
+            following steps MUST be performed:
+          <list style="symbols">
+            <t>First, the type of the SessionDescription is checked against the
+                  current state of the PeerConnection:
+              <list style="symbols">
+                <t>If the type is "offer", the PeerConnection state MUST be
+                      either "stable" or "have-remote-offer".</t>
+                <t>If the type is "pranswer" or "answer", the PeerConnection
+                      state MUST be either "have-local-offer" or
+                      "have-remote-pranswer".</t>
+              </list>
+            </t>
+            <t>If the type is not correct for the current state, processing MUST
+            stop and an error MUST be returned.</t>
+            <t>Next, the SessionDescription is parsed into a data structure,
+              as described in the
+              <xref target="sec.parsing-a-desc"/> section below. If parsing
+              fails for any reason, processing MUST stop and an error MUST be
+              returned.
+            </t>
+            <t>Finally, the parsed SessionDescription is applied as described in
+              the <xref target="sec.applying-a-remote-desc"/> section below.</t>
+          </list>
+        </t>
+      </section>
 
-      <section title="Applying a Local Description" anchor="sec.applying-a-local-descript"></section>
+      <section title="Parsing a Session Description" anchor="sec.parsing-a-desc">
+        <t>When a SessionDescription of any type is supplied to
+        setLocal/RemoteDescription, the implementation must parse it and
+        reject it if it is invalid. The exact details of this
+        process are explained below.</t>
+        <t>The SDP contained in the session description object consists of a
+        sequence of text lines, each containing a key-value expression,
+        as described in <xref target="RFC4566"/>, Section 5.
+        The SDP is read,
+        line-by-line, and converted to a data structure that contains the
+        deserialized information. However, SDP allows many types of lines, not
+        all of which are relevant to JSEP applications, and the ones that are
+        not relevant can be ignored.
```

Comment:
I'd remove the part about not relevant can be ignored here and instead just deal with it when we get to the specifics. When we get to the specifics, I would phrase it more as not JSEP was changing the definiton of SDP but more in the example of the k= lines is describes a SDP situation that is now allowed in the WebRTC profile of RTP security thus a browser will never generate and any SDP that contained it would not be a capability the browser had thus by it's definition it would be ignored.